doc/manual/decoding.rst

```diff
@@ -27,6 +27,11 @@ Scikit-learn API enforces the requirement that data arrays must be 2D. A common
 
 To recover the original 3D data, an ``inverse_transform`` can be used. The ``epochs_vectorizer`` is particularly useful when constructing a pipeline object (used mainly for parameter search and cross validation). The ``epochs_vectorizer`` is the first estimator in the pipeline enabling estimators downstream to be more advanced estimators implemented in Scikit-learn. 
 
+Vectorizer
```

Comment:
I have a feeling that you should start with `EpochsVectorizer` and make minimal changes to it (e.g., type-checking etc.) so as to make this work. But I am not up to date with API discussion which has taken place with @kingjr . But just putting it out there :)